### DISCUSSION TITLE
Skyboxes come back?

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#3896] [Tech Support / Q&A] [answered]
When you added the 2d and 3d skybox again, any specific date?

**Comment by vrad-exe** [comment]
#3891

**Comment by CN1o2** [comment]
I read practically everything, but I can't find anything that says a specific date for their return

**Comment by vrad-exe** [comment, marked as answer]
There isn't one, the feature hasn't even started being worked on yet.

**Comment by CN1o2** [comment]
Well, thanks anyway.
